src/index.js

```diff
@@ -1,8 +1,12 @@
 import './style.css';
 import addTask from './modules/addTask.js';
-import { addTaskItem, getTask, updateTask } from './modules/storeTask.js';
+import {
+  addTaskItem, deleteTask, getTask, updateTask,
+} from './modules/storeTask.js';
+import handleCompleted from './modules/handleCompleted.js';
 import refresh from './assets/refresh.png';
 import enter from './assets/enter.png';
+import removeTask from './modules/removeTask';
```

Comment:
Please kindly fix this `import` error in `src/index.js`
![image](https://github.com/over-geek/Todo-List/assets/78251927/85eb93c6-ff92-4f96-b1b5-f8c9798c1778)

Expected 🤓✔️ 
```JavaScript
import removeTask from './modules/removeTask.js';
```